content-src/actions/action-manager.js

```diff
@@ -53,6 +53,9 @@ function RequestExpect(type, expect, options = {}) {
   if (options.query) {
```

Comment:
Can you actually delete the `options.query`as well, now that we've decided not the use it?